src/com/mostc/pftt/util/HostEnvUtil.java

```diff
@@ -51,6 +51,10 @@
 		= "https://aka.ms/vs/15/release/VC_redist.x86.exe";
 	static final String Link_VC15_Redist_X64
 		= "https://aka.ms/vs/15/release/VC_redist.x64.exe";
+	static final String Link_VC16_Redist_X86
```

Comment:
It's `VS16`, not `VC16`. It's going to be renamed for the PHP packages as well, so please keep that naming here, too.

Thanks.

Reply:
I'm a bit confused here... I thought Visual C++ redistributable package is the only dependency that PHP takes, then the VC** naming convention make more sense. Does PHP take any dependency from Visual Studio other then VC?

Reply:
For the Windows PHP packages we used to use the *toolset*'s major version number prefixed by "vc". That worked fine until Visual Studio 2017 which didn't raise the major version number compared to 2015. Unfortunately, a VS 2015 pre-release did, so the packages got named "vc15" and we stuck with it to minimize confusion. Since VS 2017 didn't increase the toolset's major number either, we decided to go with the internal version number of VS, but to change the prefix to "vs" to clarify that this number no longer reflects the toolset version. And although the redistributables are about the C part only, it seems to be sensible to choose the same naming convention as for the Windows PHP packages.

Reply:
I see. So we will keep VC15 and start using VS from VS16.